analytics/terraform/spark-k8s-operator/helm-values/yunikorn-values.yaml

```diff
@@ -107,6 +107,8 @@ yunikornDefaults:
   queues.yaml: |
     partitions:
       - name: default
+        nodesortpolicy:
+          type: binpacking
```

Comment:
This is very important feature: Add some details to explain this feature

Reply:
added a quick call out in the values file and link to the policy docs. 

I didn't find a great place on the site to add a callout, maybe we should add a section on Yunikorn and some of the benefits/config we have?